packages/kata-sandbox/src/deployment.ts

```
@@ -0,0 +1,270 @@
+        environmentId: event.environmentId,
+        endpoint: event.endpoint,
+        workspaceRoot: state.intent.workspaceRoot,
+        kataHome: state.intent.kataHome,
+        identifiedAt: event.identifiedAt,
+      };
+    case "Deleted":
+      if (state === undefined) {
+        throw fail("missing-deployment", "The deployment does not exist.");
+      }
+      const deploymentId =
+        state.state === "Deleted" ? state.deploymentId : state.intent.deploymentId;
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)